client/js/utils.js

```diff
@@ -58,6 +59,15 @@ function forceFocus() {
 	input.trigger("click").trigger("focus");
 }
 
+// Reusable scrollIntoView parameters for channel list / user list
```

Comment:
This works if I step through it using a debugger breakpoint, but does not appear to actually scroll if I let the page run. But given other little things that don't work quite right in my dev environment, I pushed it anyway to let others with proper dev setups test it.